daemon/logger/journald/journald.go

```diff
@@ -88,7 +88,7 @@ func sanitizeKeyMod(s string) string {
 			v = '_'
 		}
 		// If (n == "" && v == '_'), then we will skip as this is the beginning with '_'
-		if !(n == "" && v == '_') {
+		if n != "" || v != '_' {
```

Comment:
Looks like the comment no longer matches the logic here. We could decide to use a `continue` for this one; it's probably slightly more comprehensible;

```go
if n == "" && v == '_' {
	// skip leading underscores
	continue
}
n += string(v)
```